cpu/stm32/include/periph/f7/periph_cpu.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_stm32
+ * @{
+ *
+ * @file
+ * @brief           STM32F7 CPU specific definitions for internal peripheral handling
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ *
+ */
+
+#ifndef PERIPH_F7_PERIPH_CPU_H
+#define PERIPH_F7_PERIPH_CPU_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Starting address of the ROM bootloader
+ *          see application note AN2606
+ */
+#define STM32_LOADER_ADDR   (0x1FF0EDBE)
```

Comment:
For STM32F72xxx/73xxx, STM32F74xxx/75xxx, STM32F76xxx/77xxx it is `0x1FF00000`. In fact, I couldn't find that address for any F7 in AN2606...

Reply:
I accidentally dropped this when rebasing and then must have looked into the wrong table yesterday… fixed.